bodhi/server/templates/release.html

```diff
@@ -26,7 +26,15 @@ <h2 class="pull-left">${release['long_name']}</h2>
             % endfor
           ]
         }
-        var ctx = document.getElementById("releasechart").getContext("2d");
```

Comment:
I think this line is necessary, since ctx is used below on line 38. Also, without this line the chart is not displayed when I view http://localhost:6543/releases/EL-6 but with it the chart does appear.